social-messenger/src/App.js

```diff
@@ -57,27 +57,46 @@ const App = () => {
     return () => window.removeEventListener('message', handleThemeChange);
   }, []);
 
+  useEffect(() => {
+    const mobileChannelList = document.querySelector('#mobile-channel-list');
+    if (isMobileNavVisible) {
+      mobileChannelList.classList.add('show');
+      document.body.style.overflow = 'hidden';
+    } else {
+      mobileChannelList.classList.remove('show');
+      document.body.style.overflow = 'auto';
+    }
+
+  }, [isMobileNavVisible]);
+
+  const toggleMobile = () => {
+    const isVisible = isMobileNavVisible;
```

Comment:
do we need this variable assignment? can we just do `setMobileNav(prevVisible => !prevVisible)`